api/db-cdr/generate-cdr/cb-criteria-refactor/make-bq-prep-concept-relationship-merged.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
I had this thought as well to combine everything that is supposed to be run in sequence. Erik said something along the lines of if each file is doing something distinct, it may make sense to keep them separate. Might make things easier to debug if they are in smaller chunks as well. If you want me to combine them, I can.

Reply:
@freemabd I agree with Bill. Some of the smaller scripts at the beginning could probably be combined, but the clean up scripts at the end are large and distinct enough that they should probably be kept separate to make them easier to maintain.

Reply:
`concept-merged` and `concept-relationship-merged` are probably good candidates to be combined, but the 3 scripts at the end are kind of long, so it may make sense to keep them separated.